.github/workflows/ci.yml

```diff
@@ -667,7 +667,6 @@ jobs:
     - name: package installs
       run: |
         sudo apt-get update
-        sudo apt-get -yq install meson pkg-config gnutls-bin libnss3-tools libnss3-dev libsofthsm2 opensc expect
```

Comment:
Can't this entire step be dropped if no packages are being installed?

Reply:
Good point, let me fix it quickly.